linkml/utils/generator.py

```diff
@@ -625,6 +625,15 @@ def decorator(f: Command) -> Command:
                    default=g.valid_formats[0]))
         f.params.append(
             Option(("--metadata/--no-metadata", ), default=True, help="Include metadata in output (default=--metadata)"))
+        f.params.append(
```

Comment:
oh I see what you mean.... but these don't need to go here. Just annotate the click options here

```python
@shared_arguments(JavaGenerator)
@click.command()
def cli(yamlfile, head=True, genmeta=False, classvars=True, slots=True, **args):
    """Generate java classes to represent a LinkML model"""
    print(JavaGenerator(yamlfile, emit_metadata=head, genmeta=genmeta, gen_classvars=classvars, gen_slots=slots,  **args).serialize(emit_metadata=head, **args))
```

Reply:
That makes much more sense!